### ISSUE TITLE
After entering username/password nothing happens

### ISSUE THREAD

**Issue by colinwilke** [jheling/freeathome#122]
Uploaded and included the files and I do see the extension in Home Assistant, it also detects my installation correctly (correct IP), however when asked for username and password nothing happens. If I enter a non-existing username it says user not found so it must be somehow connecting. However there is no message when entering a wrong password. The wheel just spins indefinitely… any idea? Much appreciated.

**Comment by n3m3sis1983** [comment]
Same here. Running latest 2022.2.1 HA on a QNAP NAS. F@H access point is at firmware 3.0

**Comment by jheling** [comment]
Hello, Sorry for the late reaction. Can you set the logger level in the configuration.yaml to info? After restart of Home Assistant you should see extra logging in Configuration/Setting/Logs, Load Full Home Assistant Log.   Can you show the last entries of freeathome?

**Comment by n3m3sis1983** [comment]
![image](https://user-images.githubusercontent.com/98467984/153276856-f6ec39ce-3bf2-4071-a128-ed983a665eaf.png)

**Comment by colinwilke** [comment]
Here we go, thanks a lot for your efforts:

2022-02-09 21:25:53 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration freeathome which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-09 21:26:08 INFO (MainThread) [homeassistant.bootstrap] Setting up stage 2: {'map', 'script', 'input_select', 'tts', 'usb', 'system_health', 'mobile_app', 'media_source', 'blueprint', 'counter', 'energy', 'network', 'scene', 'input_boolean', 'dhcp', 'logbook', 'ffmpeg', 'input_datetime', 'camera', 'input_button', 'homekit', 'timer', 'zone', 'default_config', 'updater', 'sun', 'diagnostics', 'input_text', 'history', 'tag', 'upnp', 'my', 'unifiprotect', 'rpi_power', 'zeroconf', 'automation', 'trace', 'ssdp', 'group', 'met', 'input_number', 'freeathome'}
2022-02-09 21:26:13 INFO (MainThread) [homeassistant.setup] Setting up freeathome
2022-02-09 21:26:13 INFO (MainThread) [homeassistant.setup] Setup of domain freeathome took 0.0 seconds
2022-02-09 21:27:20 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
2022-02-09 21:28:20 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
2022-02-09 21:29:21 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
2022-02-09 21:30:21 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
File "/config/custom_components/freeathome/config_flow.py", line 133, in async_step_import
File "/config/custom_components/freeathome/config_flow.py", line 93, in async_step_user
File "/config/custom_components/freeathome/fah/settings.py", line 21, in load_json
2022-02-09 21:31:21 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
2022-02-09 21:32:21 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
2022-02-09 21:33:21 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
2022-02-09 21:34:21 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
2022-02-09 21:35:21 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: freeathome
2022-02-09 21:37:01 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that accessed discovery_info.get('name') instead of discovery_info.name; this will fail in version 2022.6. Please report issue to the custom component author for freeathome using this method at custom_components/freeathome/config_flow.py, line 50: if not zeroconf_info.get("name", "").startswith("free@home"):
2022-02-09 21:37:01 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that accessed discovery_info['name'] instead of discovery_info.name; this will fail in version 2022.6. Please report issue to the custom component author for freeathome using this method at custom_components/freeathome/config_flow.py, line 53: friendly_name = zeroconf_info["name"].split(":",1)[1].split(".",1)[0]
2022-02-09 21:37:01 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that accessed discovery_info['host'] instead of discovery_info.host; this will fail in version 2022.6. Please report issue to the custom component author for freeathome using this method at custom_components/freeathome/config_flow.py, line 55: await self.async_set_unique_id(zeroconf_info["host"])
2022-02-09 21:37:01 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that accessed discovery_info['host'] instead of discovery_info.host; this will fail in version 2022.6. Please report issue to the custom component author for freeathome using this method at custom_components/freeathome/config_flow.py, line 60: CONF_HOST: zeroconf_info["host"],
2022-02-09 21:40:15 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] Connect Free@Home 11c04590-6c41-4029-a96e-5a95e32d1be8@busch-jaeger.de
2022-02-09 21:40:15 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] version: 3.0.0
2022-02-09 21:40:16 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:17 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:18 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:19 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:20 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:21 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:22 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:23 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:24 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:25 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:26 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:27 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:28 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:29 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:30 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:31 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:32 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:33 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection
2022-02-09 21:40:34 INFO (MainThread) [custom_components.freeathome.fah.pfreeathome] waiting for connection

**Comment by jheling** [comment]
This is very strange. The component is stuck at the login. I'm still on an older version of the firmware, but a number of users have confirmed that it still works with the 3.00 firmware. @Tho85, do you have a clue?

**Comment by colinwilke** [comment]
Maybe it is related to the combination of SysAp 1 and firmware 3.0?

**Comment by Tho85** [comment]
@colinwilke @n3m3sis1983 Could you try to setup the integration manually, i.e. click "Add integration" in the bottom right corner, search for free@home, and enter the details there.

I suspect that something is off with the hostname advertised through zeroconf. Maybe we better switch this to the announced IP address, just to be sure.

**Comment by Tho85** [comment]
> If I enter a non-existing username it says user not found so it must be somehow connecting. However there is no message when entering a wrong password.

Yeah, the implementation is a bit strange there. We first check the available user names from a list obtained via HTTP from the SysAP, and afterwards connect via XMPP to validate the password. This may be the reason why testing for the username works, but not for the password.

**Comment by eic1cr** [comment]
I had the same logs some time ago. I also have the older SysAP 1.0. and the latest firmware 3.0 and i can confirm that it runs very well.
Mayby this could help you:
https://github.com/jheling/freeathome/issues/95#issue-848296427

**Comment by Tho85** [comment]
I don't understand why this error occurs. As far as I can see, it only concerns SysAP 1.0 with firmware 3.0, right? It would be great if someone with knowledge of Wireshark could have a look why it seems to be impossible to start an XMPP connection. I have a SysAP that works with the latest firmware, so I can't really dig into this...
